docs/papers/utils/minipaper_cracking_symmetry.py

```
@@ -0,0 +1,353 @@
+
+        # Highlight tiles
+        i_max, j_max = Z.shape[0] - 1, Z.shape[1] - 1
+        for i in range(i_max):
+            for j in range(j_max):
+                quad_z = [Z[i, j], Z[i+1, j], Z[i, j+1], Z[i+1, j+1]]
+                if all(np.abs(z - z_opt) <= highlight_delta for z in quad_z):
+                    
+                    if any(z < z_opt for z in quad_z):
+                        color = 'red'
+                    else:
+                        color = 'yellow'
```

Ruff (0.11.9)

Use ternary operator `color = 'red' if any(z < z_opt for z in quad_z) else 'yellow'` instead of `if`-`else`-block

Replace `if`-`else`-block with `color = 'red' if any(z < z_opt for z in quad_z) else 'yellow'`

(SIM108)